src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/UnitTests/Controllers/AdminControllerTests.cs

```diff
@@ -0,0 +1,208 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DevAdventCalendarCompetition.Controllers;
+using DevAdventCalendarCompetition.Models.Test;
+using DevAdventCalendarCompetition.Repository.Models;
+using DevAdventCalendarCompetition.Resources;
+using DevAdventCalendarCompetition.Services.Interfaces;
+using DevAdventCalendarCompetition.Services.Models;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Moq;
+using Xunit;
+using static DevAdventCalendarCompetition.Tests.TestHelper;
+
+namespace DevAdventCalendarCompetition.Tests.UnitTests.Controllers
+{
+    public class AdminControllerTests
+    {
+        private readonly Mock<ITestService> _testServiceMock;
+        private readonly Mock<IAdminService> _adminServiceMock;
+
+        public AdminControllerTests()
```

Comment:
to get access to _AdminController_ we need to have special privileges (_Admin_ role) - we should test that too